<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>499</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>769</width>
    <height>499</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Main</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Simulation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Animation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Visualization</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solution</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="pageMain">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QGroupBox" name="mainSettings_groupBox">
           <property name="title">
            <string>Main settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="1" column="0">
             <widget class="QLabel" name="integrationMethod_label">
              <property name="text">
               <string>Integration method</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="endTime_label">
              <property name="text">
               <string>End time, s</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="endTime">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="analysisTypeComboBox">
              <item>
               <property name="text">
                <string>Kinematic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dynamic</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="analysisType_label">
              <property name="text">
               <string>Analysis type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="integrationMethodComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Euler</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Runge-Kutta</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageSimulation">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="1" column="0">
          <widget class="QGroupBox" name="simulationSettings_groupBox">
           <property name="title">
            <string>Simulation settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="3" column="2">
             <widget class="QLineEdit" name="relTol">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="TOL_dq_i">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="TOL_C_label">
              <property name="text">
               <string>TOL C(q, t)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="Hmin">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="absTol_label">
              <property name="text">
               <string>AbsTol</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="relTolLabel">
              <property name="text">
               <string>RelTol</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="absTol">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="TOL_dq_i_label">
              <property name="text">
               <string>TOL dq_i</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="Hmin_label">
              <property name="text">
               <string>Hmin</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="TOL_C">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="Hmax_label">
              <property name="text">
               <string>Hmax</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="Hmax">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="simulation_label">
           <property name="text">
            <string>Simulation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageAnimation">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="animationSettings_groupBox">
           <property name="title">
            <string>Animation settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="3" column="2">
             <widget class="QLineEdit" name="dt_lineEdit"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="updateDisplay_label">
              <property name="text">
               <string>During simulation update display on every</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="playbackSpeed_label">
              <property name="text">
               <string>Playback speed</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="steps_lineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="steps_label">
              <property name="text">
               <string>step</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="updateDisplay_comboBox">
              <item>
               <property name="text">
                <string>dt</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>step</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="dt_label">
              <property name="text">
               <string>dt</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QDoubleSpinBox" name="playbackSpeed_doubleSpinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="keyboardTracking">
               <bool>true</bool>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="fps_label">
              <property name="text">
               <string>Frames per second (for video file)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QSpinBox" name="fps_spinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Animation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageVisualization">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="visualization_label">
           <property name="text">
            <string>Visualization</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QGroupBox" name="visualization_groupBox">
           <property name="title">
            <string>Open GL</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="2">
             <widget class="QPushButton" name="editBackgroundColor_pushButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="backgroundColor_lineEdit">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="updateDisplayStep">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Background color</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="updateDisplayStepLabel">
              <property name="text">
               <string>Update display on i-th stp</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="infoDisplay_groupBox">
           <property name="title">
            <string>Info display</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="filename_checkBox">
              <property name="text">
               <string>Filename</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="simulationTime_checkBox">
              <property name="text">
               <string>Simulation time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="simulationStepNumber_checkBox">
              <property name="text">
               <string>Step number</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="timeAndDate_checkBox">
              <property name="text">
               <string>Time and date</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="GCS_groupBox">
           <property name="title">
            <string>GCS </string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QCheckBox" name="showGCS_checkBox">
              <property name="text">
               <string>Show GCS</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="showGCS_checkBox_2">
              <property name="text">
               <string>Show GCS View</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="solutionSettings_page">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QLabel" name="solution_label">
           <property name="text">
            <string>Solution settings</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="whenSimulationIsFinished_groupBox">
           <property name="title">
            <string>When simulation is finished</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="restoreInitialConditionsStatus">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Restore initial conditions</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="loadSolutionFileStatus">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Load solution</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="solutionFiletypes_label">
              <property name="text">
               <string>Solution filetype</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="solutuionFiletypes_comboBox">
              <item>
               <property name="text">
                <string>.sol</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>.csv</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>.xlsx</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
